dds/DCPS/transport/framework/TransportRegistry.cpp

```diff
@@ -696,29 +714,35 @@ TransportRegistry::create_transport_template_instance(DDS::DomainId_t domain, co
               }
               int val = 0;
               if (!convertToInteger(custom, val)) {
-              ACE_ERROR_RETURN((LM_ERROR,
-                        ACE_TEXT("(%P|%t) ERROR: TransportRegistry::")
-                        ACE_TEXT("create_transport_template_instance ")
-                        ACE_TEXT("could not convert %s to integer\n"),
-                        custom.c_str()),
-                       -1);
+                ACE_ERROR_RETURN((LM_ERROR,
+                                  ACE_TEXT("(%P|%t) ERROR: TransportRegistry::")
+                                  ACE_TEXT("create_transport_template_instance ")
+                                  ACE_TEXT("could not convert %s to integer\n"),
```

Comment:
Hmm, a bunch of %s for OPENDDS_STRING.c_str() in here too. I don't know that it really hurts anything, but %C is the tighter formatting match.